src/DSP.jl

```diff
@@ -24,6 +21,14 @@ end
 
 @importffts
 
+if VERSION >= v"0.7.0-DEV.602"
+    if isdefined(Base, :DSP)
+        import Base: conv, conv2, deconv, filt, filt!, xcorr
+    else
+        export conv, conv2, deconv, filt, filt!, xcorr
+    end
+end
+
 include("dspbase.jl")
```

Comment:
Sorry for the delay, been pretty absent on GitHub due to JuliaCon. I'll try that fix now.